examples/hello_world/proto/hello_world.proto

```diff
@@ -30,6 +30,4 @@ message HelloResponse {
 service HelloWorld {
   rpc SayHello(HelloRequest) returns (HelloResponse);
   rpc LotsOfReplies(HelloRequest) returns (stream HelloResponse);
-  rpc LotsOfGreetings(stream HelloRequest) returns (HelloResponse);
-  rpc BidiHello(stream HelloRequest) returns (stream HelloResponse);
```

Comment:
I think we should keep these around, since they match the upstream service definition, and may be useful / used in other parts of the codebase. Case in point, they are already used by the split grpc experiment @conradgrobler is working on.